### ISSUE TITLE
question about exchange class inheritance

### ISSUE THREAD

**Issue by npomfret** [ccxt/ccxt#5114]
I've been trying to add some API calls to a the bluetrade class but have run into some problems due to the fact it inherits from bittrex.

I want to mix v3 api calls into the current implementation.  The problem is that currently there is a version set, so all urls get a `v2` added to their path.

If I remove the version and add `v2/` to each api path then the super class fails because the method names are wrong.

@kroitor do you have any suggestions for a _nice_ way to work around this problem?

**Comment by npomfret** [comment]
I really struggled with bleutrade.  I've bodged a new api call to a v3 endpoint just in an attempt to get something working.  I ended up giving bleutrade it's own sign method... hope I didn't break anything.  

I really don't think the inheritance pattern is helpful in the case.  The sign method in bittrex is particularly complicated.  I think someone has gone to a lot of effort in order to avoid a bit of duplication.  But the result is a bit of a tangled mess.  IMO this is a case where more code and a little duplication (i.e. one `sign` method per class) is a better way forward because the current solution is hard (for me) to understand.  For example, it took me about an hour to figure out that the http method name (get/post) was being added to the url path.
